resteasy-links/src/test/java/org/jboss/resteasy/links/test/Book.java

```diff
@@ -13,7 +13,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-@Mapped(namespaceMap = @XmlNsMap(jsonName = "atom", namespace = "http://www.w3.org/2005/Atom"))
+//@Mapped(namespaceMap = @XmlNsMap(jsonName = "atom", namespace = "http://www.w3.org/2005/Atom"))
```

Comment:
Can we have a TODO or FIXME comment here with a reference to the JIRA for the 2nd part of this effort?